tests/app/helpers/test_template_helpers.py

```diff
@@ -1,12 +1,86 @@
 
 
-def test_get_footer_context_nisra_theme(app):
+def test_footer_context_census_theme(app):
+    with app.app_context():
+        expected = {
+            "lang": "en",
+            "crest": True,
+            "newTabWarning": "The following links open in a new tab",
+            "copyrightDeclaration": {
+                "copyright": "Crown copyright and database rights 2020 OS 100019153.",
+                "text": "Use of address data is subject to the terms and conditions.",
+            },
+            "rows": [
+                {
+                    "itemsList": [
+                        {
+                            "text": "Help",
+                            "url": "https://census.gov.uk/help/how-to-answer-questions/online-questions-help/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "Contact us",
+                            "url": "https://census.gov.uk/contact-us/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "Languages",
+                            "url": "https://census.gov.uk/help/languages-and-accessibility/languages/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "BSL and audio videos",
+                            "url": "https://census.gov.uk/help/languages-and-accessibility/accessibility/accessible-videos-with-bsl/",
+                            "target": "_blank",
+                        },
+                    ]
+                }
+            ],
+            "legal": [
+                {
+                    "itemsList": [
+                        {
+                            "text": "Cookies",
+                            "url": "https://census.gov.uk/cookies/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "Accessibility statement",
+                            "url": "https://census.gov.uk/accessibility-statement/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "Privacy and data protection",
+                            "url": "https://census.gov.uk/privacy-and-data-protection/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "Terms and conditions",
+                            "url": "https://census.gov.uk/terms-and-conditions/",
+                            "target": "_blank",
+                        },
+                    ]
+                }
+            ],
+        }
+
+        survey_config = get_survey_config(
+            theme="census",
+            language="en",
+        )
```

Comment:
`get_survey_config` should be tested independently of `ContextHelper`. For this test `ContextHelper` can be passed a direct instantiation of the `SurveyConfig` class. This applies to the other tests that do this as well.

Reply:
👍🏻

Reply:
Done